charts/redpanda/values.yaml

```diff
@@ -58,27 +58,25 @@ rackAwareness:
   # this if you have your own custom Node annotation to use instead.
   nodeAnnotation: topology.kubernetes.io/zone
 
-#
 # Authentication
 auth:
-  #
   # SASL configuration
   sasl:
-    enabled: false
+    enabled: true
+    # if unspecified, mechanism defaults to SCRAM-SHA-512
+    # mechanism: SCRAM-SHA-512
     # user list
-    # TODO create user at startup
+    # The chart assumes that the first user is "admin"
```

Comment:
What do you mean by "admin"?

Redpanda has only super-users notion. Any other service account needs to have ACL set up.

Reply:
thank you for clarifying that, It wasnt clear to me.

Reply:
Updated the text to account for this.